docs/performanceprofile/performance_profile.md

```diff
@@ -180,5 +180,6 @@ WorkloadHints defines the set of upper level flags for different type of workloa
 | ----- | ----------- | ------ | -------- |
 | highPowerConsumption | HighPowerConsumption defines if the node should be configured in high power consumption mode. The flag will affect the power consumption but will improve the CPUs latency. | *bool | false |
 | realTime | RealTime defines if the node should be configured for the real time workload. | *bool | false |
+| perPodPowerManagement | perPodPowerManagement defines if the node should be configured in per pod power management. | *bool | false |
```

Comment:
Should this mention that perPodPowerManagement is not compatible with highPowerConsumption?

Reply:
I'm not sure if we should include this in the API or in the docs or release notes @MarSik

Reply:
I can see we have a test that states they cannot be added together. maybe worth adding a note here